### DISCUSSION TITLE
小米首款2億畫素鏡頭手機 12T Pro 正式發表

### DISCUSSION THREAD

**Discussion by ABCDEFGHIJKLMNO12345678** [1101-GS4719-C/Python-Community-GS4719-C#739] [General]
小米可說是相當大眾的品牌 不僅價格平易近人 更有著高cp 雖說不像蘋果那樣吸引了 但在價格方面可說是讓人愛不釋手 最近要推出的新品搭載著2億畫素 價格還不到20000 拍出之畫值甚至可以媲美相機了

https://tw.news.yahoo.com/%E5%B0%8F%E7%B1%B3%E9%A6%96%E6%AC%BE2%E5%84%84%E7%95%AB%E7%B4%A0%E9%8F%A1%E9%A0%AD%E6%89%8B%E6%A9%9F-12t-pro-%E6%AD%A3%E5%BC%8F%E7%99%BC%E8%A1%A8-010432616.html

**Comment by WEI-CHEN-CHUN** [comment]
高畫素相對考驗軟硬體的處理效率，且畫素越高會占用更多空間

**Comment by AnAnistired** [comment]
其實畫素不代表畫質，看iphone的祖傳鏡頭或是pixel的超強軟體就知道了（話說兩億畫素這樣照片不會佔更多記憶體嗎？

**Comment by Doreen331** [comment]
除非安卓手機真的有什麼很大或很實用的新技術是蘋果沒有的，否則不論信仰是否存在，一般的蘋果用戶應該很難跳槽，雖然小米手機畫素比較高，但若我個人選擇還是會選iphone...

**Comment by williamyen0623** [comment]
身為狗粉的我覺得小米的手機其實不是那麼有競爭力，但其他產品是真的好用，像是螢幕燈、手環等等

**Comment by Pietra1226** [comment]
用過蘋果真的覺得其他的UI設計的不是很流暢，但如果性能匹敵蘋果，我可能會考慮
